### PR TITLE
[DEVHUB-1929] Update copy for books

### PR DESCRIPTION
**Pull request by mcolella14**
## Jira Ticket:

[DEVHUB-1929](https://jira.mongodb.org/browse/DEVHUB-1929)

### CODE REVIEW

File: src/pages/books.tsx [hunk 1]
```diff
@@ -375,7 +375,7 @@ export const getStaticProps: GetStaticProps<{
                 'Rajesh Nair',
                 'Rachelle Palmer',
             ],
-            publishedDate: 'Sept. 2023',
+            publishedDateText: 'will be published in Nov. 2023',
```
Comment (jackyw2018) [inline src/pages/books.tsx:378]: Should we capitalize "will"?

**Review by jackyw2018 — APPROVED** [review]
LGTM with a minor comment

File: src/pages/books.tsx (hunk 1 above)
Comment (mcolella14) [reply src/pages/books.tsx:378]: I can clarify with Liz, but this is the wording/capitalization that was quoted in the ticket.

Comment (mcolella14) [reply src/pages/books.tsx:378]: @jackyw2018 Ok Liz said to capitalize it, thanks for catching!

**Pull request merged**